lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -4802,6 +4894,7 @@ class ConvertTorchToTosa : public ConvertTorchToTosaBase<ConvertTorchToTosa> {
   target.addIllegalOp<AtenOp>();                                               \
   patterns.add<ConvertAtenFillScalarOp<AtenOp>>(typeConverter, context);
     INSERT_FILL_SCALAR_PATTERN(AtenFill_ScalarOp);
+    INSERT_FILL_SCALAR_PATTERN(AtenFillScalarOp);
```

Comment:
Is this related to the pass?

Reply:
Yes, every Empty comes with a fill, this is just legalizing the `fill` using the same implementation as the `fill_` op